releasenotes/notes/0.12/add-NoiseModel-from_backendproperties-1a3d6d976133a661.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    Added a new class method :meth:`~.NoiseModel.from_backend_properties` to
+    the :class:`NoiseModel`. This enables constructing a new :class:`~.NoiseModel`
+    from a :class:`~.BackendProperties` object. Similar functionality used
```

Comment:
`.BackendProperties` won't correctly link if that's a Terra object and we're aiming for intersphinx lookups.  I don't believe intersphinx will do the suffix match.  If we want to link to a Terra object, we usually have to do `` :class:`~qiskit.providers.BackendProperties` `` (or whatever its complete Terra documentation name is).  That said, I've never tried the explicit intersphinx form `` :external+qiskit:class:`.BackendProperties` `` - that _might_ work.

Reply:
I'll just update this to be the full path, that's easiest.

Reply:
Done in: https://github.com/Qiskit/qiskit-aer/pull/1739/commits/b6f8e606bd36bbe66cda1246f748515a6a203c3d